docs/neo-n3/dapp_development/transactions.md

```diff
@@ -1,87 +1,52 @@
 builder.signers(AccountSigner.calledByEntry(account));
 ```
 
-Signers are associated with a witness scope that restricts how the signer's witness* can be used in an invocation.
-For example, if a signer is only needed for paying the transaction fee, the witness scope can be set to *None*. There
-are two signer classes to be aware of - the `AccountSigner` and the `ContractSigner`. Use `AccountSigner` for signers
-that are backed by an account. It provides static builder methods for all witness scopes.
+Signers are associated with a witness scope that restricts how the signer's witness* can be used during invocation. For instance, if a signer is only needed for paying the transaction fee, the witness scope can be set to *None\*. There are two signer classes to be aware of - `AccountSigner` and `ContractSigner`. Use `AccountSigner` for signers backed by an account, which provides static builder methods for all witness scopes.
 
-Use `ContractSigner` if the signer is a smart contract. This kind of signer doesn't require a signature, but will call
-the contract's `verify(...)` method when the transaction is executed. Since contract signers cannot pay the transaction
-fees, the class `ContractSigner` does not provide a builder method for the *None* witness scope. Further, the
-`ContractSigner`'s builder methods can take contract parameters in case the contract's `verify(...)` method has extra
-parameters.
+Use `ContractSigner` if the signer is a smart contract. This type of signer does not require a signature but calls the contract's `verify(...)` method during transaction execution. Since contract signers cannot pay transaction fees, the `ContractSigner` class does not provide a builder method for the _None_ witness scope. Furthermore, the `ContractSigner`'s builder methods can accept contract parameters if the contract's `verify(...)` method has extra parameters.
 
-**Note:** A transaction requires at least one `AccountSigner` that will pay for the fees.
+**Note:** A transaction requires at least one `AccountSigner` to pay for the fees.
 
-> *A witness is a script pair that contains an invocation and a verification script. Both scripts are simply a
-> sequence of NeoVM instructions. In the basic case of an account signer, the invocation script consists of the
-> instruction to push the signature data on the NeoVM stack, whereas in the verification script, the corresponding
-> public key data is pushed on the NeoVM stack followed by an instruction to check its signature provided in the
-> invocation script.
+> \*A witness is a script pair that consists of an invocation and a verification script. Both scripts are sequences of NeoVM instructions. In the case of an account signer, the invocation script includes instructions to push the signature data onto the NeoVM stack. In the verification script, the corresponding public key data is pushed onto the NeoVM stack, followed by an instruction to verify the signature provided in the invocation script.
```

Comment:
What's the purpose of `\*`?

I think that we wanted to make this bold or italic back in the day, but maybe it got lost over time (with some docs updates). No?